wallet-new/src/Cardano/Wallet/Kernel/Transactions.hs

```diff
@@ -570,6 +570,7 @@ data SignTransactionError =
     SignTransactionMissingKey Address
   | SignTransactionErrorUnknownAddress Address
   | SignTransactionErrorNotOwned Address
+  | SignTransactionUnableForExternalWallet
```

Comment:
Not sure we'll need this. It'd be better if we enforce that only "classic" wallets can be signed.